tree/tree/inc/TTree.h

```diff
@@ -439,7 +445,7 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
    virtual Long64_t        GetSelectedRows() { return GetPlayer()->GetSelectedRows(); }
    virtual Int_t           GetTimerInterval() const { return fTimerInterval; }
            TBuffer*        GetTransientBuffer(Int_t size);
-   virtual Long64_t        GetTotBytes() const { return fTotBytes; }
+   virtual Long64_t        GetTotBytes() const { std::lock_guard<std::mutex> sentry(fCounterMutex); return fTotBytes; }
```

Comment:
For correct operation, shouldn't you also force the assignment to a local variable:
std::lock_guard<std::mutex> sentry(fCounterMutex); 
auto retValue = fTotBytes;
return retValue;